source/reference/api/start.txt

```diff
@@ -325,6 +338,27 @@ Response
    :language: json
    :copyable: false
 
+Example: Start Sync to Remap Namespaces
```

Comment:
Can we move this block to the namespace remapping page? Just for consistency with how we documented document filtering [source/reference/beta-program/document-filtering.txt](https://github.com/mongodb/docs-cluster-to-cluster-sync/pull/319/files#diff-5bf60d842bdc5a11f8ff6610849c6db52cb038106736e7f6e82cf34dbd547913)

Reply:
Fixed